tests/snapd

```diff
@@ -87,55 +79,22 @@ lxc profile device add default root disk pool=default path=/
 lxc network create lxdbr0
 lxc profile device add default eth0 nic nictype=bridged parent=lxdbr0 name=eth0
 
-snap set lxd ceph.builtin=true
-
 systemctl restart snap.lxd.daemon.service
 
-for _ in $(seq 60); do
-    lxd waitready --timeout=10 >/dev/null 2>&1 && break
-done
+lxd waitready --timeout=300
 
 # Check that user access works
-if [ "${ID}" != "ubuntu-core" ]; then
-    (
-        set -e
-        usermod -G lxd -a jenkins || true
-        LXC=$(which lxc)
-        su jenkins -c "${LXC} info"
-    )
-fi
+set -e
```

Comment:
This `set -e` is probably not wanted as it would prevent catching any error but that of the last command in the script.

Reply:
Removed